### ISSUE TITLE
Add support for following redirects

### ISSUE THREAD

**Issue by addshore** [JeroenDeDauw/QueryrAPI#23]
Currently this thing has no idea / concept of redirects.
They are important and should be accounted for.

**Comment by JeroenDeDauw** [comment]
Good point. At present the EntityStore component does not support redirects, and neither does the importing code (Replicator and JsonDumpReader). Actually not sure how redirects are represented in the dump, if at all.

**Comment by addshore** [comment]
They are not represented in the dump :/
Thought there is a bug for that
https://phabricator.wikimedia.org/T98320

**Comment by JeroenDeDauw** [comment]
What about the API? I know it follows redirects by default, but will it somehow tell you it did, so you can also store the redirect info?

And is there a sane way to fetch all redirect info, to mitigate it not being in the dump?

**Comment by addshore** [comment]
https://www.wikidata.org/w/api.php?action=query&list=allredirects&arnamespace=0&arprop=ids|title&arlimit=100

Just ignore non items

Also youll have to do a second query to find the target from the pageid

**Comment by JeroenDeDauw** [comment]
If I'm not mistaken, some smug created 500k redirects or something, so doing this via the API will take some time :)

**Comment by addshore** [comment]
Hah, yes indeed, some smug did.
[done.txt](https://github.com/JeroenDeDauw/QueryrAPI/files/45299/done.txt)
[redirects.txt](https://github.com/JeroenDeDauw/QueryrAPI/files/45296/redirects.txt)
